content/events/2024-02-26-hackathon-image-analysis-in-galaxy/index.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Hackathon Image Analysis in Galaxy
+date: '2024-02-26'
+days: 5
+tease: "This hackathon addresses the improvement of Galaxy for image analysis"
+continent: EU
+location:
+  name: "University of Freiburg, Germany"
+contact: "Beatriz Serrano-Solano, Yi Sun, Björn Grüning"
+tags: [hackathon, imaging]
+subsites: [all]
+---
+
+# General information
+
+- **Date**: Feb 26th - Mar 3rd 2024 (full week)
+- **Format**: hybrid
+- **Location**: Seminar room at Georges-Köhler-Allee 79, 79104 Freiburg
+- **Time**: full day, 9/10am CEST to open end
+- **Cost**: Free
+- **Contact**: Beatriz Serrano-Solano ([beatriz.serrano.solano@eurobioimaging.eu](mailto:beatriz.serrano.solano@eurobioimaging.eu)), Yi Sun ([yi.sun@embl.de](mailto:yi.sun@embl.de), Björn Grüning ([gruening@informatik.uni-freiburg.de](mailto:gruening@informatik.uni-freiburg.de)
+
+This event addresses individuals interested in hacking Galaxy to improve analyzing image data and is organized by [Euro-BioImaging](https://www.eurobioimaging.eu) and [NFDI4BIOIMAGE](https://nfdi4bioimage.de). 
+
+# Registration
+
+Registration is free, but the space is limited, so we recommend to register soon. Please wait for our confirmation before booking your trip.
+
+Please note that we expect participants to cover their own travel and accommodation costs. If this is not possible, please let us know, and we will do our best to find a solution.
+
+As this is a hybrid event, remote attendance is also possible (details down below).
+
+<div class="row justify-content-center">
+  <a class="btn w-25 btn-dark" href="https://docs.google.com/forms/d/1nw8xvXGojgI3DUr3tyC9lmAs7kunkf_aVLOgUwRs1VI" role="button" target="_blank">Register now</a>
+</div>
+
+# Remote attendance
+
+Hacking is not restricted to onsite, we appreciate your participation from anywhere else, too. We provide a Zoom connection and involve a Meeting Owl to include you as close as possible.
+
+**Connection details for online attendance:**
+
+https://embl-org.zoom.us/j/95969566635?pwd=czB1dHNxejVNbm90dlg2cmVSMnZsQT09 
```

Comment:
I'd also avoid publishing this, it's on the agenda. We can link the [agenda](https://docs.google.com/document/d/1pVToHXfwDLCkRZF0ifg9aVeT3cg-cjGEYWVSyIRd2hc/edit#heading=h.84pge7bu74zm) somewhere